extra/modules/confiant-ad-quality/src/main/resources/module-config/confiant-ad-quality.yaml

```diff
@@ -17,3 +17,4 @@ hooks:
         long-interval-attempts: 336
         long-interval: 1800000
       scan-state-check-interval: 600000
+      bidders-to-exclude-from-scan:
```

Comment:
do we need to provide an example
is this comma separated?

Reply:
or the example is here: https://github.com/prebid/prebid-server-java/pull/2670/files#diff-1bb08eaea30ab0ffe869994b44c347ca6ea1d13cf5bab5dc2a4ab0a91cbfd6faR77

Reply:
Right. Example is inside **README.md** file. And separately will be added to the official Prebid docs repo.